ulib/axlibc/src/unistd.rs

```diff
@@ -27,3 +35,68 @@ pub unsafe extern "C" fn abort() -> ! {
 pub unsafe extern "C" fn exit(exit_code: c_int) -> ! {
     sys_exit(exit_code)
 }
+
+/// Sleep for a number of seconds.
+#[cfg(feature = "signal")]
+#[no_mangle]
+pub unsafe extern "C" fn alarm(seconds: c_uint) -> c_uint {
+    let it = ctypes::itimerval {
+        it_interval: ctypes::timeval {
```

Comment:
```
it_interval: ctypes::timeval::default();
```
By the way, please check whether setting `it_interval` to zero is ok? I find that it is not set to zero for `ualarm` in the following code.

Reply:
`it.it_value.tv_sec` should be param `seconds`. It is inconvenient to use default.